### PR TITLE
Warning about antivirus software after benchmark failure

### PR DESCRIPTION
**Pull request by lukasz-pyrzyk**
According to the #364 

In case of invalid error code, analyzer prints warning about installed antivirus software and InProcessToolchain. 

Tested with Symantec and Windows Defender.

### CODE REVIEW

File: src/BenchmarkDotNet.Core/Analysers/EnvironmentAnalyser.cs [hunk 1]
```diff
@@ -24,6 +29,28 @@ public override IEnumerable<Conclusion> AnalyseSummary(Summary summary)
...
+        }
+
+        private static string CreateWarningAboutAntivirus(ICollection<Antivirus> avProducts)
+        {
+            var sb = new StringBuilder("Detected error exit code from one of the benchmarks. " +
+                                       "It might be caused by following antivirus software:");
+            sb.AppendLine();
+
+            foreach (var av in avProducts)
+                sb.AppendLine($"\t - {av}");
```
Comment (AndreyAkinshin) [inline src/BenchmarkDotNet.Core/Analysers/EnvironmentAnalyser.cs]: Could we use spaces instead of `\t`?

**Comment by AndreyAkinshin** [comment]
It looks much better now.
Could you also add a note about it in `docs/guide/FAQ.md`?

File: src/BenchmarkDotNet.Core/Analysers/EnvironmentAnalyser.cs (hunk 1 above)
Comment (lukasz-pyrzyk) [inline src/BenchmarkDotNet.Core/Analysers/EnvironmentAnalyser.cs]: Fixed

**Comment by lukasz-pyrzyk** [comment]
Changed to spaces and added a FAQ note. What do you think?

**Comment by AndreyAkinshin** [comment]
Great, thanks!

**Comment by mbjasonayer** [comment]
My AV is controlled by my work's IT so disabling AV is out for me. Does anyone have a work around for this issue? I tried a suggestion from another thread about running Command Prompt with Administrator rights from the executable's location, but that did not help.

**Pull request merged**